virttest/libvirt_storage.py

```diff
@@ -184,6 +184,24 @@ def pool_state(self, name):
                 pass
         return None
 
+    def pool_autostart(self, name):
+        """
+        Get pool's autostart.
+
+        :return: yes/no, and None when something wrong.
+        """
+        try:
+            pools = self.list_pools()
+        except process.CmdError:
+            return None
+        if self.pool_exists(name):
+            details_dict = pools[name]
+            try:
+                return details_dict['Autostart']
+            except KeyError:
+                pass
+        return None
```

Comment:
```
try:
    return self.list_pools()[name]["Autostart"]
except process.CmdError, KeyError:
    return None
```

Reply:
Is it for more simple or something else?

Reply:
`pool_exists` also calls `list_pools()`, this redundant calls can be avoided.

Reply:
Agreed and updated code. beside, pool_state is updated in this way also.